package.json

```diff
@@ -24,11 +24,17 @@
     "update": "ncu -u"
   },
   "dependencies": {
+    "@mdx-js/mdx": "^1.6.22",
+    "@mdx-js/react": "^1.6.22",
+    "@reach/router": "^1.3.4",
```

Comment:
reach router and it's types are not needed to be installed, they comes with gatsby. You can imort them from gatsby itself

Reply:
So we're keeping reach router !